server.babel.js

```diff
@@ -1,2 +1,2 @@
-require('babel/register');  
+require('babel-register');
```

Comment:
This should be `require('babel-core/register')`. Also needed updating in dbSetup.babel.js. Have updated myself.